packages/libnghttp2.rb

```diff
@@ -30,21 +28,20 @@ class Libnghttp2 < Package
   depends_on 'py3_cython' => :build
 
   def self.build
-    Dir.mkdir 'builddir'
+    FileUtils.mkdir 'builddir'
     Dir.chdir 'builddir' do
       system "cmake -G Ninja #{CREW_CMAKE_OPTIONS} \
-        -DENABLE_EXAMPLES=OFF \
         -DENABLE_SHARED_LIB=ON \
         -DENABLE_STATIC_LIB=ON .."
       system 'samu'
     end
   end
 
-  def self.check
-    system 'samu -C builddir test'
-  end
-
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
   end
+
+  def self.check
+    system 'samu -C builddir test'
+  end
```

Comment:
I actually prefer the check above the install since it follows the workflow in sequence from top to bottom. :)

Reply:
Reasonable.